app/assets/stylesheets/application.scss

```diff
@@ -59,6 +59,11 @@ html, body {
   background-color: white;
 }
 
+@media (max-width: 992px)
```

Comment:
Is the media query necessary? I tried testing with and without it and couldn't find the difference

Reply:
It is actually only for desktop users as there is this fixed height in the wrapper. It would be better to work with "viewheight" instead of percentage.

Reply:
Just coming back around to this now. The media query would better be wrapped in something specific for the "Can't create rooms" view and moved to the `rooms.scss`. Something like this:

```
#cant-create-room-wrapper {
  @media....
}
```